tests/ttnn/unit_tests/operations/test_new_conv2d.py

```diff
@@ -40,6 +42,31 @@ def _nearest_32(x):
 #     plt.close()
 
 
+def write_to_file_special(file_name, tensor):
```

Comment:
these two new functions don't seem to be used, and their purpose is not obvious.
Remove randomly added commented code throughout the file